subprojects/tooling-api/src/main/java/org/gradle/tooling/internal/protocol/events/InternalJvmTestDescriptor.java

```diff
@@ -53,4 +53,11 @@
      */
     String getMethodName();
 
+    /**
+     * Returns the name of the included build, if any.
+     *
+     * @return The name of the included build, can be null.
+     * @since 6.8
```

Comment:
If this is internal API, there should be no `@since` annotation.

Reply:
Actually, I've realized that we only need to declare the method on the concrete class (i.e. on `DefaultTestDescriptor`).